pkg/operators/wasm/helpers.go

```diff
@@ -101,3 +98,14 @@ func isDataArrayHandle(handle uint32) bool {
 func getIndexFromDataArrayHandle(dataHandle uint32) int {
 	return int(dataHandle &^ dataArrayHandleFlag >> 16)
 }
+
+func (i *wasmOperatorInstance) writeToDstBuf(src []byte, dst uint64) (uint64, error) {
```

Comment:
This function is not necessary: it is the same as `writeToDstBuffer`, except that it returns the uint64. But the return value is never used. So you could keep `writeToDstBuffer`.

Reply:
ohman! It was me trying to split the logic into two different commits. I removed `writeToDstBuf`